src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -21,13 +26,31 @@ const SubscriptionsUtilizedCard = lazy(() => import('../../SmartComponents/Subsc
 const PatchManagerCard = lazy(() => import('../../SmartComponents/PatchManager/PatchManagerCard'));
 const RemediationsCard = lazy(() => import('../../SmartComponents/Remediations/RemediationsCard'));
 
-const Dashboard = () => {
-
+const Dashboard = ({ inventorySummary, workloads, fetchInventory, inventoryFetchStatus }) => {
     const permission = useContext(PermissionContext);
     const intl = useIntl();
+    const [supportsSap, setSupportsSap] = useState(true);
+
+    const fetchInventoryFn = useCallback(() => {
```

Comment:
Instead of fetching the full inventory information, you can query `/api/inventory/v1/system_profile/sap_system` and check the count for `value: true` result.

Reply:
But that requires a bit more code to be done (reducer, action, etc.)

Reply:
I left out the tags off the query for that very reason, but i agree, not super thrilled using the same endpoint, will checkout the other one yah mentioned! <3

Reply:
cuz it also means i don't need the system count 😜

Reply:
also means we can do it (the call) once n never again (till component is destroyed n recreated) rather than ever time the workplace is swapped